src/wallet/coinselection.h

```diff
@@ -193,6 +193,17 @@ struct SelectionResult
      *  hit a specific target.
      */
     CAmount input_fees{0};
+    /** Whether the input values for calculations should be the the effective value (true) or normal value (false) */
+    bool m_use_effective{true};
+    /** The algorithm used supports making change outputs. */
+    bool m_make_change{false};
+    /** The cost of making a change output and spending it in the future. Since this is largely a static parameter independent of the selection, it is not cleared by Clear() */
```

Comment:
What do you mean with "independent of the selection"? Do you mean that it's the same across different algorithms, independent of the inputs that get picked, or smth else? It's definitely dependent on the selection parameters, especially the feerate.

Reply:
Independent of the algorithm.

Reply:
Clarified the comment.